geomstats/geometry/fiber_bundle.py

```diff
@@ -23,42 +21,30 @@ class FiberBundle(Manifold, ABC):
 
     Parameters
     ----------
+    dim : int
+        Dimension of the base manifold.
     total_space : Manifold
         Total space of the bundle.
-    base : Manifold
-        Base manifold of the bundle.
-        Optional. Default : None.
     group : LieGroup
         Group that acts on the total space by the right.
         Optional. Default : None.
         Either the group or the group action must be given.
-    total_space_metric : RiemannianMetric
-        Metric to use in the total space.
-        Optional. The `metric` attribute of the total space is used if no
-        ambient metric is passed.
     group_action : callable
         Right group action. It must take as input a point of the total space
         and an element of the group, and return a point of the total space.
-    dim : int
-        Dimension of the base manifold.
-        Optional. If available the dimension of the base manifold is used,
-        or the difference between the dimension of the total space and the
-        group. Either dim, base or group must be given as input.
+
     """
 
     def __init__(
         self,
-        dim: int,
-        group: LieGroup = None,
-        total_space_metric: RiemannianMetric = None,
+        group=None,
         group_action=None,
         group_dim=None,
-        **kwargs
+        equip=True,
```

Comment:
The option to equip or not seems a bit odd to me as some methods in this class require a metric (mathematically) to be defined. This is the case of `align` and `horizontal_projection`. The idea behind this class was that the total space was always equipped. Could this be enforced? Maybe there should not be a kwarg `equip` here or it should always be True.

Reply:
Yes, I will enforce this.